index.js

```diff
@@ -25,7 +25,14 @@ module.exports = {
                     const u = t.getElementsByTagName(s)[0];
                     u.parentNode.insertBefore(r, u);
                 }(window, document, 'https://sc-static.net/scevent.min.js'));
-                snaptr('init', pixelId);
+                if(email){
+                    snaptr('init', pixelId,{
+                        user_email: email
+                    });
+                }else{
+                    snaptr('init', pixelId);
+                }
+                
```

Comment:
Naming is not correct. If you want to be able to pass any optional data, you don't name it only one parameter's name. 
I'll push a fix for that